### PR TITLE
Fixes: #2614 (Allow "webidl" css class on pre tags to denote a WebIDL block)

### PR DESCRIPTION
**Pull request by ridhishjain**
webidl.css need not be changed as previously `pre.idl` class was importing webidl.css which now both `pre.idl` and `pre.webidl` will be importing.
Relevant code in src/core/webidl.js

`const cssPromise = loadStyle();`
`async function loadStyle() {`
  `try {`
    `return (await import("text!../../assets/webidl.css")).default;`
  `} catch {`
    `return fetchAsset("webidl.css");`
  `}`
`}`

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -351,11 +350,16 @@ async function loadStyle() {
 }
 
 export async function run() {
-  const idls = document.querySelectorAll("pre.idl");
```
Comment (sidvishnoi) [inline src/core/webidl.js]: You can combine selectors like:
``` js
const idls = document.querySelectorAll("pre.idl, pre.webidl");
```

File: src/core/webidl.js [hunk 2]
```diff
@@ -351,11 +351,11 @@ async function loadStyle() {
...
-  if (!idls.length) {
-    return;
-  }
-  if (!document.querySelector(".idl:not(pre)")) {
+  const idls = document.querySelectorAll("pre.idl, pre.webidl");
+  if (
+    !document.querySelector(".idl:not(pre)") ||
```
Comment (sidvishnoi) [inline src/core/webidl.js]: Similarly here:
``` js
!document.querySelector(".idl:not(pre), .webidl:not(pre)")
```
This is a good read on combining CSS selectors: https://codenart.github.io/css/2017/08/24/css-9-combining-selectors.html

**Comment by ridhishjain** [comment]
@marcoscaceres review

**Comment by ridhishjain** [comment]
@marcoscaceres review, please

**Comment by ridhishjain** [comment]
The test is not passing, I can't figure out why? 
The test added is wrong I guess.

**Comment by ridhishjain** [comment]
@marcoscaceres require a little help here

**Comment by ridhishjain** [comment]
@marcoscaceres @sidvishnoi how about this?

File: tests/spec/core/webidl-spec.js [hunk 3]
```diff
@@ -1490,4 +1490,11 @@ callback CallBack = Z? (X x, optional Y y, /*trivia*/ optional Z z);
     expect(idl2.classList).toContain("respec-offending-element");
     expect(idl2.title).toContain("Optional dictionary");
   });
+  it("checks that webidl is processed same as idl", async () => {
+    const target = doc.getElementById("dict-webidl");
+    const text = "dictionary Test {};";
```
Comment (marcoscaceres) [inline tests/spec/core/webidl-spec.js]: Probably don't need this one.

File: tests/spec/core/webidl-spec.js [hunk 4]
```diff
@@ -1490,4 +1490,11 @@ callback CallBack = Z? (X x, optional Y y, /*trivia*/ optional Z z);
...
+    expect(target.textContent).toBe(text);
+    expect(target.querySelectorAll(".idlDictionary").length).toBe(1);
+    expect(target.querySelector(".idlID").textContent).toBe("Test");
```
Comment (marcoscaceres) [inline tests/spec/core/webidl-spec.js]: This is good.

File: tests/spec/core/webidl.html [hunk 5]
```diff
@@ -505,6 +505,12 @@ <h2>Dictionaries</h2>
       <pre id='dict-basic' class='idl'>
         dictionary SuperStar {};
       </pre>
+      <p>
```
Comment (marcoscaceres) [inline tests/spec/core/webidl.html]: I prefer if you kept the HTML in the other test file. Makes the tests easier to work with.

**Comment by marcoscaceres** [comment]
@ridhishjain just so you know, when filing an issue, you can create a "fenced code block" by putting three ` ```js ` backticks together, and then another three afterwards. Have a look at some handy examples here:
https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

File: tests/spec/core/webidl.html (hunk 5 above)
Comment (ridhishjain) [reply tests/spec/core/webidl.html]: means u want me to use 
```js
const body = `
  <pre class='webidl' id='dict-webidl'>
        dictionary Test {};
  </pre>
`;
const ops = makeStandardOps(null, body);
const doc = await makeRSDoc(ops);
const target = doc.getElementById("dict-webidl");
```
in `webidl-spec.js`?

**Comment by ridhishjain** [comment]
@marcoscaceres is this ok?

**Pull request merged**